gm4_balloon_animals/data/gm4_balloon_animals/tags/entity_types/balloon_animal.json

```diff
@@ -0,0 +1,22 @@
+{
+  "values": [
+    "minecraft:bee",
+    "minecraft:camel",
+    "minecraft:chicken",
+    "minecraft:cow",
+    "minecraft:donkey",
+    "minecraft:fox",
+    "minecraft:frog",
+    "minecraft:goat",
+    "minecraft:horse",
+    "minecraft:llama",
+    "minecraft:ocelot",
+    "minecraft:panda",
+    "minecraft:parrot",
+    "minecraft:pig",
+    "minecraft:rabbit",
+    "minecraft:sheep",
+    "minecraft:turtle",
+    "minecraft:wolf"
+  ]
+}
```

Comment:
This was manually collected and sorted.  Should it be generated by beet, @Bloo-dev ?

Reply:
Yeah, probably a good idea. Then the entire module is based ont he csv!
Make sure to add the `minecraft:` prefix to the csv.